src/main/java/core/basesyntax/DrawFigure.java

```diff
@@ -0,0 +1,5 @@
+package core.basesyntax;
+
+public interface DrawFigure {
```

Comment:
Think about why we split abstract class and interface.
```suggestion
public interface Drawable {
```